kingpin/bin/deploy.py

```diff
@@ -120,7 +120,7 @@ def main():
     if args.build_only:
         try:
             get_main_actor(dry=False)
-        except actor_exceptions.ActorException as e:
+        except Exception as e:
```

Comment:
Because if something raises an exception that wasn't caught and converted to a kingpin exception then this code doesn't catch it and exits with status 0.

For example - yaml parser raised ScannerException which was not caught by anything